app/src/main/java/com/example/wangyf/android_dokumi_test/Main2Activity.kt

```diff
@@ -2,11 +2,21 @@ package com.example.wangyf.android_dokumi_test
 
 import android.support.v7.app.AppCompatActivity
 import android.os.Bundle
+import android.preference.PreferenceManager
 
 class Main2Activity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main2)
+
+        if (true) {
+            val i: Int? = null
+            i!!.toString()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wang6793453/android-dokumi-test/blob/796c54f31b33c5c7e98ae2b17503807cd43d917c/app/src/main/java/com/example/wangyf/android_dokumi_test/Main2Activity.kt#L15"></span>Return value of String.valueOf(int) ignored in com.example.wangyf.android_dokumi_test.Main2Activity.onCreate(Bundle)</td>
    </tr>
  </tbody>
</table>